### DISCUSSION TITLE
Value

### DISCUSSION THREAD

**Discussion by Flonixcorn** [Comfy-Org/ComfyUI#235] [Ideas]
Value nodes for easy setting of image sizes with math nodes for multiplication please! also: bools, quick loads, model and lora searching by name, snap to grid, hiding of connections/nodes, hotkeys etc.

**Comment by Flonixcorn** [comment]
also panning with middle mouse for easier use

**Comment by WASasquatch** [comment]
Be sure to search the community for ideas already suggest a lot.
